test/rulesets/LinearAlgebra/norm.jl

```diff
@@ -39,6 +39,26 @@
             @test extern(rrule(fnorm, zero(x))[2](ȳ)[2]) ≈ zero(x)
             @test rrule(fnorm, x)[2](Zero())[2] isa Zero
         end
+        ndims(x) > 1 && @testset "non-strided" begin
+            xp = if x isa Matrix
+                view(x, [1,2,3], 1:3)
+            elseif x isa Array{T,3}
+                PermutedDimsArray(x, (1,2,3))
+            end
+            @test !(xp isa StridedArray)
+            y = fnorm(x)
+            # ẋ = rand(T, size(xp)) # rand_tangent(xp)
+            x̄ = rand(T, size(xp)) # rand_tangent(xp)
+            ȳ = rand_tangent(y)
+            # frule_test(fnorm, (xp, ẋ))
+            rrule_test(fnorm, ȳ, (xp, x̄))
+        end
+        T == Float64 && ndims(x) == 1 && @testset "Integer input" begin
+            x = [1,2,3]
+            _, int_back = rrule(fnorm, x)
+            _, float_back = rrule(fnorm, float(x))
```

Comment:
Probably tested elsewhere as well, but perhaps wouldn't hurt to test the results for the forward pass here as well.

Reply:
Done